msg.go

```diff
@@ -369,6 +370,15 @@ Loop:
 					}
 				}
 			}
+			// never exceed the allowed label count lenght (63) 
+			if labels >= 63 { 
+				return "", lenmsg, &Error{err: "name exceeds 63 labels"}
```

Comment:
There's no such limit in DNS; [RFC 1035](https://tools.ietf.org/html/rfc1035#section-2.3.4) requires the wire-format length of _each_ label to be 63 octets or less, which is already guaranteed by the `c & 0xC0` check.